js/src/Ice/TcpTransceiver.js

```diff
@@ -122,6 +122,19 @@ if (typeof net.createConnection === "function") {
                 return;
             }
 
+            try {
+                this._fd.end();
```

Comment:
sends a FIN packet, see https://nodejs.org/api/net.html#socketenddata-encoding-callback

Reply:
What exceptions can this function throw?

Reply:
You are right, no need for try/catch `.end()` doesn't throw